libcontainer/exeseal/cloned_binary_linux.go

```diff
@@ -47,8 +47,6 @@ func sealMemfd(f **os.File) error {
 	// errors because they are not needed and we want to continue
 	// to work on older kernels.
 	fd := (*f).Fd()
-	// F_SEAL_FUTURE_WRITE -- Linux 5.1
-	_, _ = unix.FcntlInt(fd, unix.F_ADD_SEALS, unix.F_SEAL_FUTURE_WRITE)
```

Comment:
Wouldn't hurt to keep a comment saying we don't use `F_SEAL_FUTURE_WRITE` for a reason, but it's okay without it as well. Up to you.

Reply:
done